grizzly_extras/types.py

```diff
@@ -0,0 +1,9 @@
+from typing import Tuple, Optional
+
+
+class PermutationVectored:
```

Comment:
Maybe some comment/documentation on what the class does / what the bools stand for ?

Reply:
![MicrosoftTeams-image](https://user-images.githubusercontent.com/815510/176171408-0c72ec31-cc26-4861-95d2-80586dbbfe96.png)